### ISSUE TITLE
Encoding 1st column of new compound sort key 

### ISSUE THREAD

**Issue by samarthwhistle** [awslabs/amazon-redshift-utils#389]
When using ColumnEncodingUtility with `--new-sort-keys` option, 1st column in new sort keys is encoded automatically.

For example, here's the original table definition:
```
create table public."logfile"(
"event_ts" timestamp   NOT NULL encode RAW 
,"device_sn" varchar(10)  NOT NULL encode RAW
,"fw_version" varchar(256)   encode RAW 
,"filename" varchar(256)   encode RAW 
)
SORTKEY(fw_version);
```
Running the utility with `--new-dist-key device_sn` and `--new-sort-keys device_sn,event_ts` options would generate this definition:
```
create table public."logfile"(
"event_ts" timestamp   NOT NULL encode zstd 
,"device_sn" varchar(10)  NOT NULL encode zstd DISTKEY
,"fw_version" varchar(256)   encode RAW 
,"filename" varchar(256)   encode zstd 
)
SORTKEY(device_sn,event_ts);
```
I see two concerns with this:
1. columns in compound sort key are encoded which is against what is suggested in [Top 10 Performance Tuning Techniques](https://aws.amazon.com/blogs/big-data/top-10-performance-tuning-techniques-for-amazon-redshift/)
2. fw_version column which is no longer a sort key isn't encoded 

Thank you!

**Comment by IanMeyers** [comment]
This is now fixed in 80c40fe

**Comment by samarthwhistle** [comment]
@IanMeyers Thanks for addressing this!
